src/jv.c

```diff
@@ -1862,6 +1862,63 @@ jv jv_object_iter_value(jv object, int iter) {
 /*
  * Memory management
  */
+jv jv_new(jv input){
```

Comment:
If the point of the function is to make a deep copy that don't share any memory with the input i think it probably should have a more clear name (`jv_copy_deep` etc?) and also a comment clarifying it.

Reply:
@emanuele6 proposed `jv_clone()`. I am quite open to different names.

Reply:
Mm seem to be how rust uses copy/clone, but i'm always confusing them :) a very clear name could be something like `jv_unshare`

Reply:
Hmm, I think proposed `deepclone` like in Java if I proposed anything at all; `jv_clone` would also be confusing because in Java clone only copies one level

Reply:
then I guess `jv_unshare()` would be the best option. Then again apparently words don't have meaning in this industry so who knows if there is some conflict there.

Reply:
@wader I've never heard `unshare` used for deepclone, but I think it's a very good name for it :-)

Reply:
:) made it up right now but mostly based on @MaxBrandtner use case for it. I think something deep copy or unshare   works. but maybe deep copy is a more well known concept?

Reply:
I really quite like that idea. I shall implement it right away